projects/widgets/toc.json

```diff
@@ -101,6 +101,12 @@
       "title": "Compatibiliteit",
       "uri": "docs/compatibiliteit.md",
       "slug": "compatibiliteit"
+    },
+    {
+      "type": "item",
+      "title": "Linken van artikels aan events via widgets",
```

Comment:
The title on the page itself is "Linken van artikels aan events via redactionele tools". Is the difference on purpose, or accidental?

Also: This page feels like it's dangling a bit under the rest (because "Compatibiliteit" feels like a good page to end with). Maybe the structure would be a bit better if we add a "Redactionele tools" divider above this page, to clearly separate it from the rest of the Widgets docs?

The JSON code for that is:

```
{
  "type": "divider",
  "title": "Redactionele tools"
}
```

Reply:
This was on purpose: with the title "Linken van artikels aan events via widgets" everybody that reads the menu item has a basic idea of what to expect, with "Redactionele tools" that is not the case.

But I understand this is not ideal

Reply:
Personally I think the "via widgets" is implied because you are browsing the Widgets documentation. So maybe we can just make it "Linken van artikels aan events" in the menu, and keep the title on the page "Linken van artikels aan events via redactionele tools"?

About the position in the menu: We don't have to add the "Redactionele tools" divider if you think that is going to be confusing/unclear. But I would try to find a better place for it in the menu then or put the "Compatibiliteit" someplace else (for example more at the top, like under "Inleiding") because the order of pages (Introduction, a list of how-to's, then "Compatibiliteit", and then suddenly another how-to) is a bit weird maybe I think.

Reply:
I went with your first suggestion!